readme.txt

```diff
@@ -3,8 +3,8 @@ Contributors: code4recovery
 Donate link: https://code4recovery.org/contribute
 Requires at least: 3.2
 Requires PHP: 5.6
-Tested up to: 6.6
-Stable tag: 3.15
+Tested up to: 6.6.1
```

Comment:
we don't need to specify patch versions on `Tested up to` - wordpress only complains on minor versions

Reply:
.1 removed but the following added on line 304:
= 3.15.1 =
* Reformat 'Change Detection Email' for better legability. [more info](https://github.com/code4recovery/12-step-meeting-list/issues/1497)

Reply:
i think we want to keep the stable tag and lose the tested up to…